src/chains/soroban/Soroban Report.md

```diff
@@ -129,7 +184,132 @@ Stellar has faced **regulatory scrutiny**, particularly regarding the classifica
+[FT SCV 1](https://stellarbeat.io/nodes/GARYGQ5F2IJEBCZJCBNPWNWVDOFK7IBOHLJKKSG2TMHDQKEEC6P4PE4V),
+[FT SCV 2](https://stellarbeat.io/nodes/GCMSM2VFZGRPTZKPH5OABHGH4F3AVS6XTNJXDGCZ3MKCOSUBH3FL6DOB),
+[FT SCV 3](https://stellarbeat.io/nodes/GA7DV63PBUUWNUFAF4GAZVXU2OZMYRATDLKTC7VTCG7AU4XUPN5VRX4A),
+[LOBSTR 1](https://stellarbeat.io/nodes/GCFONE23AB7Y6C5YZOMKUKGETPIAJA4QOYLS5VNS4JHBGKRZCPYHDLW7),
+[LOBSTR 4](https://stellarbeat.io/nodes/GA7TEPCBDQKI7JQLQ34ZURRMK44DVYCIGVXQQWNSWAEQR6KB4FMCBT7J),
+[LOBSTR 5](https://stellarbeat.io/nodes/GA5STBMV6QDXFDGD62MEHLLHZTPDI77U3PFOD2SELU5RJDHQWBR5NNK7),
+[Hercules by OG Technologies](https://stellarbeat.io/nodes/GBLJNN3AVZZPG2FYAYTYQKECNWTQYYUUY2KVFN2OUKZKBULXIXBZ4FCT))
+and running an outdated Stellar-core version
+([Alpha Node Validator](https://stellarbeat.io/nodes/GBPLJDBFZO2H7QQH7YFCH3HFT6EMC42Z2DNJ2QFROCKETAPY54V4DCZD)).
+These warnings are significant, since most organizations require agreement from
+them, Tier 1 organizations "bear the safety and
+liveness of the Stellar network on their shoulders". Notably, one of their
+validators' main responsibilities is publishing an archive of transactions in
+order to ["make the network more
+resilient"](https://developers.stellar.org/docs/validators/tier-1-orgs).
+
+All validators run the same software
+([stellar-core](https://github.com/stellar/stellar-core)], which highlights the
+extreme centralization of the software development process around Stellar's core
+protocol. Consequently, bugs in stellar-core can escalate to systemic risks and
+affect the entire Stellar network.
+
+### Soroban Security
+
+Soroban's core contracts were [audited by Veridise between October-December
+2023](https://veridise.com/wp-content/uploads/2024/11/VAR_Stellar_Soroban-5.pdf).
+The review discovered 8 issues, of which one was of medium severity (fixed).
+
+On March 2024 the SDF launched the [Soroban Security Audit
+Bank](https://stellar.org/blog/developers/the-soroban-audit-bank-fostering-a-secure-smart-contract-ecosystem),
+a program for funding security audits of Soroban-based projects. As of March
+2025 it is unclear how many audits and tools have received funding from this
+program.
+
+Various tools exist to assist Soroban smart contract developers to write secure
+contracts or formally verify their implementations. Among others, they include:
+
+- [CoinFabrik's Scout](https://github.com/CoinFabrik/scout-soroban);
+- [Certora's formal Wasm tool](https://www.certora.com/blog/formally-verifying-webassembly);
+- [OtterSec's stellar-verify](https://github.com/otter-sec/stellar-verify);
+- [Runtime Verification's Komet](https://docs.runtimeverification.com/komet).
+
+The existence of such tools enables the creation of high-assurance code and can
+help increase confidence in the security guarantees of Soroban-based smart
+contracts.
+
+---  
+
+## **Section 4: Axelar Integration Components**
+
```

Comment:
1. "[Axelar Amplifier Stellar](https://github.com/axelarnetwork/axelar-amplifier-stellar)" -> "[`axelar-amplifier-stellar`](https://github.com/axelarnetwork/axelar-amplifier-stellar)" (x2)
2. "Axelar Cross-chain Gateway Protocol" -> "Axelar Cross-Chain Gateway Protocol"
3. "developed in Rust programming language" -> "developed in the Rust programming language"
4. "following components" -> "following smart contracts"